src/main/groovy/org/labkey/gradle/task/DoThenSetup.groovy

```diff
@@ -217,6 +217,10 @@ class DoThenSetup extends DefaultTask
                         }
             }
 
+        new File("${project.tomcat.catalinaHome}/lib/jtds.jar").delete();
+        new File("${project.tomcat.catalinaHome}/lib/mysql.jar").delete();
+        new File("${project.tomcat.catalinaHome}/lib/postgresql.jar").delete();
```

Comment:
I assume you've done this with the files not available and not seen it cause problems?  Since ServerDeploy also captures these (and other) jar files in the TOMCAT_LIB_UNVERSIONED_JARS list, we might do well to put this into a separate list we could iterate through:
```
public static final List<String> JDBC_JARS = ['jtds.jar', 'mysql.jar', 'postgres.jar'];
public static final List<String> TOMCAT_LIB_UNVERSIONED_JARS = JDBC_JARS + [...others...]
...
JDBC_JARS.each(String name -> {String name ->
           new File("${project.tomcat.catalinaHome}/lib/${name}").delete()
})
```

Reply:
Yes, seems to work fine whether those files exist or not.

Reply:
I like your change. Are you suggesting that the delete itself happens in ServerDeploy or in the same place in DoThenSetup?

Reply:
Change made... seems to work